src/Tasks/Microsoft.NET.Build.Tasks/build/platforms/Windows.targets

```diff
@@ -0,0 +1,23 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(_ShortFrameworkIdentifier)' == 'uap'">
+    <NugetTargetMoniker>UAP,Version=v10.0</NugetTargetMoniker>
+    <CopyLocalLockFileAssemblies Condition="'$(CopyLocalLockFileAssemblies)' == ''">false</CopyLocalLockFileAssemblies>
+    <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
+    <DefineConstants Condition="'$(DisableImplicitFrameworkDefines)' != 'true'">$(DefineConstants);NETFX_CORE;WINDOWS_UWP</DefineConstants>
+    <TargetPlatformVersion Condition="'$(TargetPlatformVersion)' == ''">10.0.10240.0</TargetPlatformVersion>
+    <TargetPlatformMinVersion Condition="'$(TargetPlatformMinVersion)' == ''">10.0.10240.0</TargetPlatformMinVersion>
+    <TargetFrameworkVersion>v5.0</TargetFrameworkVersion>
+    <LanguageTargets>$(MSBuildExtensionsPath)\Microsoft\WindowsXaml\v$(VisualStudioVersion)\Microsoft.Windows.UI.Xaml.$(_SdkLanguageName).targets</LanguageTargets>
```

Comment:
> These won't work at all with dotnet build. It only works with MSBuild.exe

Then I think the layering is broken here.  As it states in the [README of this repo](https://github.com/dotnet/sdk#welcome-to-dotnet-sdk):

> Core functionality needed to create .NET Core projects, **that is shared between VisualStudio and CLI**.